package.json

```diff
@@ -1,43 +1,48 @@
 {
   "name": "svg-typewriter",
-  "version": "0.3.0",
-  "description": "A library for measuring, manipulating, and writing text on SVG.",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/palantir/svg-textwriter.git"
-  },
-  "author": "Andrzej Skrodzki @ Palantir Technologies",
-  "main": "svgtypewriter.js",
-  "typings": "svgtypewriter.d.ts",
+  "version": "1.0.0",
+  "description": "SVG Text Measurement Line Wrapper",
+  "main": "build/src/index.js",
```

Comment:
Okay, we'll have to make sure we run `yarn build` or something before `npm publish` right?